tests/scale-tests/harness/harnessctl/install_components.go

```
@@ -0,0 +1,379 @@
+			warnf("%s not ready (may be disabled or named differently in this chart version): %v", d, err)
+		}
+	}
+	infof("NVSentinel installed in namespace %s", cfg.NVSNamespace)
+	return nil
+}
+
+// ensureCertManagerWebhookHealthy probes the webhook; heals expired CA if needed.
+func (c *clients) ensureCertManagerWebhookHealthy(ctx context.Context, cfg Config) error {
+	ns := cfg.CertManagerNamespace
+	if _, err := c.kube.AppsV1().Deployments(ns).Get(ctx, "cert-manager-webhook", metav1.GetOptions{}); err != nil {
+		return nil // not installed here; nothing to heal
```

golangci-lint (2.12.2)

[error] error is not nil (line 278) but it returns nil

(nilerr)